src/gui/layout/qgslayoutitemguiregistry.h

```diff
@@ -424,6 +434,13 @@ class GUI_EXPORT QgsLayoutItemGuiRegistry : public QObject
      */
     bool addItemGroup( const QgsLayoutItemGuiGroup &group );
 
+    /**
+     * Unregisters a item group from the registry. This must be done after calling
+     * removeLayoutItemGuiMetadata() for every item types associated with the group.
+     *
```

Comment:
```suggestion
     * Unregisters a item group from the registry.
     *
     * This must be done after calling
     * removeLayoutItemGuiMetadata() for every item types associated with the group.
     *
     * \since QGIS 4.0
```

Reply:
Unregistered **an** item group.
_Sorry I just read review comments_